packages/components/dropdowns/dropdown-menu/src/dropdown-menu.spec.tsx

```diff
@@ -74,4 +74,42 @@ describe('DropdownMenu', () => {
     });
     expect(await screen.findByText('Content')).not.toBeVisible();
   });
+
+  it('should pass data attributes without menuType', async () => {
+    await render(
+      <DropdownMenu
+        data-testid="dropdown-container"
+        triggerElement={
+          <SecondaryButton label="Trigger" data-testid="trigger-element" />
+        }
+      >
+        <DropdownMenu.ListMenuItem>Option 1</DropdownMenu.ListMenuItem>
```

Comment:
Will the data attribute also get passed down to the ListMenuItem when it's default DropdownMenu type?

Reply:
Yes, the data attributes would get passed down to the ListMenuItem even if its the default menuType. I have updated the test case to make it more explicit in this commit: https://github.com/commercetools/ui-kit/pull/2855/commits/22313d616d558c6ca97fc06bb8f9b2875a330f9c